tpython/lib/utils/futils.py

```diff
@@ -1,136 +1,129 @@
-from genericpath import exists
+from genericpath import isdir
 import shutil as sh
-from os.path import isfile, join
+from os.path import join
 from os import listdir
 import os
 import logging
-from frontmatter import Frontmatter as yaml
+import frontmatter
 
-def writeFile(folder,filename,contents):
+
+def initPath(path):
+    os.makedirs(path)
+
+
+def writeFile(folder, filename, contents):
```

Comment:
is the logic of this function correct now? If the folder doesn't exist then create it and write the file. What if the folder does exist but the file doesn't? Then the file will never be written. Or am I misunderstanding? Can the folder hold multiple files or is it just 1?